catalogues/repohost.txt

```diff
@@ -0,0 +1,5 @@
+GitHub
+BitBucket
+GitLab
+Codeberg
+Sourcehut
```

Comment:
maybe we can add `{any person}'s gitlab` or similar?

Reply:
What about `{silent unique mod m}{unique author[m]} makes the {unique repohost} repo for {m} private` in `drama`?